core/keyserver/keyserver.go

```diff
@@ -147,37 +158,19 @@ func (s *Server) getEntry(ctx context.Context, domainID, userID, appID string, f
 		}
 	}
 
-	// Fetch log proofs.
-	secondTreeSize := logRoot.GetSignedLogRoot().GetTreeSize()
-
-	// Consistency proof.
-	var logConsistency *tpb.GetConsistencyProofResponse
-	if firstTreeSize != 0 {
-		logConsistency, err = s.tlog.GetConsistencyProof(ctx,
-			&tpb.GetConsistencyProofRequest{
-				LogId:          domain.LogID,
-				FirstTreeSize:  firstTreeSize,
-				SecondTreeSize: secondTreeSize,
-			})
-		if err != nil {
-			glog.Errorf("tlog.GetConsistency(%v, %v, %v): %v",
-				domain.LogID, firstTreeSize, secondTreeSize, err)
-			return nil, status.Errorf(codes.Internal, "Cannot fetch log consistency proof")
-		}
-	}
-
-	// Inclusion proof.
+	// SignedMapHead to SignedLogRoot inclusion proof.
+	secondTreeSize := sth.GetTreeSize()
 	logInclusion, err := s.tlog.GetInclusionProof(ctx,
 		&tpb.GetInclusionProofRequest{
-			LogId: domain.LogID,
+			LogId: d.LogID,
 			// SignedMapRoot must be placed in the log at MapRevision.
 			// MapRevisions start at 1. Log leaves start at 1.
```

Comment:
Can you clarify meaning of "log leaves start at 1" here as leaf indices are 0 based.

Reply:
The comment is incorrect.  Updated:
`MapRevisions start at 0. Log leaves start at 0.`